althea_types/src/contact_info.rs

```diff
@@ -12,29 +12,45 @@ use phonenumber::PhoneNumber;
 pub struct ContactDetails {
     pub phone: Option<String>,
     pub email: Option<String>,
+    sequence_number: Option<u32>,
```

Comment:
so this is specifically for the exits, we don't need sequence number here because the exit always takes the latest on checkin. We'll get more into the details of this when we start to upgrade the exit format.

Reply:
This was mostly because there's a conversion function going from ContactDetails to ContactType. I've just changed the conversion function to take in sequence_number as an argument instead.